tests/scripts/github-action-helper.sh

```diff
@@ -250,6 +250,16 @@ function wait_for_prepare_pod() {
   kubectl -n rook-ceph describe deployment/rook-ceph-osd-0 || true
 }
 
+function wait_for_cleanup_pod() {
```

Comment:
Should the test fail if this just hits the 180s timeout?

Reply:
It will, `timeout` returns `1` if the code inside is still running.

Reply:
ah, of course